### PR TITLE
Extract views and flows from selenium E2E test file 

### PR DESCRIPTION
**Pull request by mminder**
## Description & Motivation
Separate views from test file to have a better overview over tests without scrolling a bunch first.

## How Has This Been Tested?
Tests have been executed locally and should now also run on CI.

## Types of changes

- [ ] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to change)

## Checklist:

- [x] My code follows the code style of this project.
- [ ] My change requires a change to the documentation.
- [ ] I have updated the documentation accordingly.
- [x] I have added tests to cover my changes.
- [x] All new and existing tests passed.

### CODE REVIEW

**Review by nmattia — COMMENTED** [review]
I'll admit that for a newbie like me, passing the driver around explicitly made things really clear; I wonder if the (class) indirection hurts clarity. Happy to go with it if you think it's a net improvement though! I'll let @kritzcreek be the judge

File: src/frontend/src/test-e2e/flows.ts [hunk 1]
```diff
@@ -0,0 +1,55 @@
...
+
+export const FLOWS = {
+  registerNewIdentity: async (
+    deviceName: string,
+    driver: ThenableWebDriver
+  ): Promise<string> => {
+    const welcomeView = new WelcomeView(driver);
+    const registerView = new RegisterView(driver);
+    const singleDeviceWarningView = new SingleDeviceWarningView(driver);
+    const recoveryMethodSelector = new RecoveryMethodSelector(driver);
```
Comment (kritzcreek) [inline src/frontend/src/test-e2e/flows.ts:18]: I'd prefer initializing these right before usage instead

**Review by kritzcreek — CHANGES_REQUESTED** [review]
Looks good in general 👍 I'd like to move the initialization of the view objects closer to their usage, like we're already doing it in the smaller tests.

File: src/frontend/src/test-e2e/flows.ts (hunk 1 above)
Comment (mminder) [reply src/frontend/src/test-e2e/flows.ts]: done

**Review by kritzcreek — APPROVED** [review]

**Pull request merged**